diesel/src/sqlite/connection/mod.rs

```diff
@@ -413,6 +428,17 @@ mod tests {
         assert_eq!(Ok(3), added);
     }
 
+    sql_function!(fn answer() -> Integer);
+
+    #[test]
+    fn register_noarg_function() {
+        let connection = SqliteConnection::establish(":memory:").unwrap();
+        answer::register_nondeterministic_impl(&connection, || 42).unwrap();
```

Comment:
I would like to see a similar test for the `register_impl` variant here for completeness.

Reply:
Fixed.